lib/bloom_net_center_client/models/txn.rb

```diff
@@ -22,5 +22,12 @@ class Txn
 
     ATTRS = self.attribute_set.map(&:name)
 
+    STATUSES = {
+      unfunded: "unfunded",
+      funded: "funded",
+      claiming: "claiming",
+      claimed: "claimed",
```

Comment:
oh, I was thinking we could use this as: `if remote_txn.status == BloomNetCenterClient::Txn::STATUSES[:funded]`  operations. Is there a better way, or a more concise way for implementing some sort of status helper?